R/loo-glossary.R

```diff
@@ -68,13 +68,37 @@
 #' proposal distribution). The Pareto k diagnostic estimates how far an
 #' individual leave-one-out distribution is from the full distribution. If
 #' leaving out an observation changes the posterior too much then importance
-#' sampling is not able to give reliable estimate. If `k<0.5`, then the
-#' corresponding component of `elpd_loo` is estimated with high accuracy.
-#' If `0.5<k<0.7` the accuracy is lower, but still ok. If `k>0.7`,
-#' then importance sampling is not able to provide useful estimate for that
-#' component/observation. Pareto k is also useful as a measure of influence of
-#' an observation. Highly influential observations have high k values. Very high
-#' k values often indicate model misspecification, outliers or mistakes in data
+#' sampling is not able to give reliable estimate. Pareto smoothing stabilizes
+#' importance sampling and guarantees finite variance estimate with a
+#' cost of some bias.
+#'
+#' The diagnostic threshold for Pareto k depends on sample size
+#' \eqn{S}. For simplicity the nominal sample size \eqn{S} is used
+#' when computing the sample size specific threshold. This is likely
+#' to provide optimistic threshold, but for many purposes this is fine
+#' if the MCMC effective sample size is not much smaller than the
+#' nominal sample size (e.g. if MCMC-ESS > S/4).
+#'
+#' * If \eqn{k < min(1 - 1 / log10(S), 0.7)}, where \eqn{S} is the
+#'   sample size PSIS estimate and the corresponding Monte
+#'   Carlo standard error estimate are reliable.
+#'
+#' * If \eqn{1 - 1 / log10(S) <= k < 0.7} PSIS estimate and the
+#'   corresponding Monte Carlo standard error estimate are not reliable,
+#'   but increasing (effective) sample size \eqn{S} above 2200 may help.
```

Comment:
Should the justification of the S = 2200 be outlined somewhere? I don't immediately see how it comes about

Reply:
I added an explanation